datahub-web-react/src/app/settings/AccessTokens.tsx

```diff
@@ -200,6 +200,7 @@ export const AccessTokens = () => {
     ];
 
     const onChangePage = (newPage: number) => {
+        window.scrollTo({ top: 0, left: 0 });
```

Comment:
since we're using this same function in all of these places, I think it would be smart to only define it once and reuse that same function everywhere. So I might suggest creating a new file called `searchUtils.ts` in `src/app/shared` where you can define a function `scrollToTop` that simply calls this window function here. That way it's more reusable and easier to read/understand what's going on here.